cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/PeerMetadata.java

```diff
@@ -27,10 +28,15 @@
   private final List<NamespaceAllocation> namespaceAllocations;
   // metadata associated with the peer. ex: project, region when peer is located
   private final Map<String, String> metadata;
+  // Text that describes this tethering
+  private final String description;
```

Comment:
do we have to support modifying the description?

Reply:
No, atleast not in the MVP release. We also don't support editing metadata and namespace allocations.